Ensures both PHPUnit and WP-CLI are available allowing a simplificati…

ChrisWiegman opened the pull request:
Ensures both PHPUnit and WP-CLI are available allowing a simplification of the development environment.

dkotter commented:
@ChrisWiegman Checked this out to test it and was getting an error when composer was trying to install wp-cli. I fixed it by changing the version from master to dev-master, which I think is correct but if not, let me know. And then I fixed the merge conflict.

If it sounds correct to be dev-master, I'll get this merged in. Thanks!

ChrisWiegman commented:
That sounds right. I hadn't received the error but that makes perfect sense. 

Sent from something really hard to type on

> On Jan 28, 2016, at 18:24, Darin Kotter <notifications@github.com> wrote:
> 
> @ChrisWiegman Checked this out to test it and was getting an error when composer was trying to install wp-cli. I fixed it by changing the version from master to dev-master, which I think is correct but if not, let me know. And then I fixed the merge conflict.
> 
> If it sounds correct to be dev-master, I'll get this merged in. Thanks!
> 
> —
> Reply to this email directly or view it on GitHub.
>
